src/_pytest/_io/saferepr.py

```diff
@@ -75,15 +90,15 @@ def safeformat(obj: object) -> str:
         return _format_repr_exception(exc, obj)
 
 
-def saferepr(obj: object, maxsize: int = 240) -> str:
+def saferepr(obj: object, maxsize: Optional[int] = 240) -> str:
```

Comment:
Is it possible to synergize this 240 with `_DEFAULT_REPR_MAX_SIZE`?

Reply:
Sure, will do!